### PR TITLE
[FIX] Darwin temporary password file access

### PR DESCRIPTION
**Pull request by patred20**
This PR changes method of "waiting" until validator starts so the temporary password could be loaded and deleted.

### CODE REVIEW

File: shell_scripts/lukso [hunk 1]
```diff
@@ -621,14 +621,20 @@ start_validator() {
...
-        echo "ERROR: Cannot start validator"
-        break;
-      fi
+      echo -n " . "
       TRIES=$((TRIES+1))
     done
+
+    if [[ ! $(pgrep -x "lukso-validator") ]]; then
+      echo "ERROR: Cannot start validator"
+      break;
```
Comment (mxmar) [inline shell_scripts/lukso]: `break;` is pointless here.

`break: only meaningful in a `for', `while', or `until' loop`

File: shell_scripts/lukso [hunk 2]
```diff
@@ -621,14 +621,20 @@ start_validator() {
...
+    else
```
Comment (mxmar) [inline shell_scripts/lukso]: I think if you exit script in `if` statement you don't need to do `if else`, just `    if [[ ! $(pgrep -x "lukso-validator") ]]; then` with `exit`.

File: shell_scripts/lukso [hunk 3]
```diff
@@ -621,14 +621,20 @@ start_validator() {
   if [[ -n $TEMPPASSFILE ]]; then
     # Wait until validator starts
     TRIES=0
-    while [[ ! $(lsof -t -i:7000) ]]; do
+    while [[ $TRIES -lt 5 ]]; do
```
Comment (mxmar) [inline shell_scripts/lukso]: I think you can prepare `const` variable at the begining of this script or just declare like:
```
##### network info
CHAIN_ID=""
NETWORK_ID=""
#####
```

**Review by mxmar — CHANGES_REQUESTED** [review]
I've left comments

**Comment by mxmar** [comment]
And please solve conflicts. `ipc` PR was merged.

File: shell_scripts/lukso (hunk 1 above)
Comment (mxmar) [reply shell_scripts/lukso]: done

File: shell_scripts/lukso (hunk 2 above)
Comment (mxmar) [reply shell_scripts/lukso]: done

File: shell_scripts/lukso (hunk 3 above)
Comment (mxmar) [reply shell_scripts/lukso]: done

**Review by sikorek — APPROVED** [review]

**Review by mxmar — APPROVED** [review]

**Pull request merged**